src/Traits/HandlesCheckout.php

```diff
@@ -213,6 +213,9 @@ public function checkoutStageStoreComplete()
             ->setUid($this->uid)
             ->process();
 
+        Session::put('paymentResponse', $paymentResponseData);
+        $this->append('paymentResponse', $paymentResponseData);
+
         // Send the payment response to the Api for processing
         $newCheckout = App::make(Checkout::class, [
```

Comment:
So we don't need to return `$newCheckout`? Should we remove the assignment to the variable then to make it more clear?

Reply:
That's right, we can just append the payment response.  So yea we don't need the $newCheckout assignment, I'll remove.